oonidata/datautils.py

```diff
@@ -18,22 +18,32 @@
     b'<meta.*?property="og:title".*?content="(.*?)"', re.IGNORECASE | re.DOTALL
 )
 
-
-def get_html_meta_title(body: bytes) -> bytes:
+def guess_decode(s: bytes) -> str:
+    """
+    best effort decoding of a string of bytes
+    """
+    for encoding in ("ascii", "utf-8", "latin1"):
+        try:
+            return s.decode(encoding)
+        except UnicodeDecodeError:
+            pass
```

Comment:
Would it be meaningful to do something like `logging.warning(...)` here? We may learn about additional encodings thanks to this warning, especially if it does not spew out too many lines.

Reply:
Yes I can add this, it could lead us to spotting interesting cases.